oci/private/versions.bzl

```diff
@@ -1,6 +1,18 @@
 "Mirror of release info"
 
 CRANE_VERSIONS = {
+    "v0.0.0-nightly": {
```

Comment:
I guess there should be a TODO to delete this as soon as they release?

Reply:
#37 will address this automatically. there is no `v0.0.0-nightly` in `google/go-containerregistry` but in `thesayyn/go-containerregistry` which is my fork. no way for me to miss this cause it won't work.

Reply:
I'll leave a TODO nevertheless.